elao.app/.manala/Dockerfile.tmpl

```diff
@@ -29,7 +33,7 @@ RUN \
     && mkdir -p /srv \
     && chmod 777 /srv \
     # User
-    && adduser --disabled-password --gecos "" docker \
+    && adduser --disabled-password --gecos "" docker --uid ${UID} \
```

Comment:
Here, the trick is to create the group just before, and use it right after for user creation.
```
&& addgroup --gid ${GID} docker \
&& adduser --disabled-password --gecos "" docker --uid ${UID} --ingroup docker \
```